src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue/PaymentsDueProcessor.cs

```diff
@@ -521,14 +466,5 @@ private void AddPaymentsDue(Provider provider, List<RequiredPayment> paymentsDue
                 LearningStartDate = earning.LearningStartDate
             });
         }
-
-        private void AddPaymentsDue(List<RequiredPayment> paymentsDue, RequiredPayment payment, decimal amountDue,
-            PeriodEarning earning, Provider provider)
-        {
-            payment.AmountDue = amountDue;
-            payment.IlrSubmissionDateTime = provider.IlrSubmissionDateTime;
-
-            paymentsDue.Add(payment);
-        }
```

Comment:
is there an argument for just removing this payments due processor code, and leave the sql still pulling through all the required fields? Not totally convinced we should, but worth discussing?